modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurators/DesignPacksProjectConfigurator.java

```diff
@@ -88,7 +92,10 @@ public void apply(Project project) {
 		GradleUtil.applyPlugin(project, LiferayBasePlugin.class);
 		GradleUtil.applyPlugin(project, NodePlugin.class);
 		GradleUtil.applyPlugin(project, ThemeBuilderPlugin.class);
-		GradleUtil.applyPlugin(project, WarPlugin.class);
+
+		_addDependenciesParentThemes(project);
+
+		_addDependenciesPortalCommonCSS(project);
```

Comment:
Hi davi. We still need these two lines code _addDependenciesParentThemes(project); _addDependenciesPortalCommonCSS(project), even we apply themeBuildPlugin. The gw build will throw "Cannot convert path to File. path='com.liferay.gradle.plugins.theme.builder.ThemeBuilderPlugin$6@5b650e78", The root cause of this is the portal style, unstyle, common css actually not installed.